### PR TITLE
refactor: improve eslint command

### PR DESCRIPTION
**Pull request by manniL**
## Type of Change

- **Something else:** Improved Linting

## What issue does this relate to?

None

### What should this PR do?
This PR improves the eslint command so it also lints the `.eslintrc` and the `nuxt.config.js`.
Automated fixes via `npm run test:eslint:fix` are included in the PR.

### Misc

Whoops, closed the PR accidentally 😅

### CODE REVIEW

File: nuxt.config.js [hunk 1]
```diff
@@ -183,15 +183,14 @@ export default async () => {
...
                     await whitelist(builder);
                 },
-                async done(builder) {
+                async done (builder) {
                     await staticFiles(builder);
                     await images(builder);
                     await fonts(builder);
-                    if (process.env.NODE_ENV && process.env.NODE_ENV === 'production')
-                        await sitemap(join(builder.nuxt.options.buildDir, 'dist', 'static'));
+                    if (process.env.NODE_ENV && process.env.NODE_ENV === 'production') { await sitemap(join(builder.nuxt.options.buildDir, 'dist', 'static')); }
```
Comment (MattIPv4) [inline nuxt.config.js:193]:
```suggestion
                    if (process.env.NODE_ENV && process.env.NODE_ENV === 'production') {
                        await sitemap(join(builder.nuxt.options.buildDir, 'dist', 'static'));
                    }
```
This line has become very long

**Review by MattIPv4 — CHANGES_REQUESTED** [review]
Otherwise, lgtm

File: nuxt.config.js (hunk 1 above)
Comment (manniL) [reply nuxt.config.js]: Yup, that's true! Applied your suggestion 😋

**Review by MattIPv4 — APPROVED** [review]
Lgtm, thanks :)

**Pull request merged**